client/src/routes/register/RegisterPage.tsx

```diff
@@ -0,0 +1,127 @@
+import type { RegisterUserReq, TokenResp } from '@backend/auth/types';
+import axios from 'axios';
+import { useContext } from 'react';
+import Button from 'react-bootstrap/Button';
+import Container from 'react-bootstrap/Container';
+import Form from 'react-bootstrap/Form';
+import { useForm } from 'react-hook-form';
+import { useNavigate } from 'react-router-dom';
+import StatusCode from 'status-code-enum';
+import { useLocalStorage } from 'usehooks-ts';
+import isEmail from 'validator/lib/isEmail';
+
+import { LocalStorageKey } from '../../constants';
+import { UserContext, useUser } from '../../contexts/user';
+import http from '../../http';
+
+type RegisterFormData = RegisterUserReq;
+
+const required = 'this field is required';
+
+const RegisterPage = () => {
```

Comment:
@Shie1ded lets add this page to the routing (see [here](https://github.com/dhyxy/busse/blob/ayalBranch/client/src/index.tsx#L16))